lib/Dialect/FIRRTL/Transforms/LowerXMR.cpp

```diff
@@ -299,8 +300,9 @@ class LowerXMRPass : public LowerXMRBase<LowerXMRPass> {
   }
 
   // Propagate the reachable RefSendOp across modules.
-  LogicalResult handleInstanceOp(InstanceOp inst) {
-    auto *mod = inst.getReferencedModule();
+  LogicalResult handleInstanceOp(InstanceOp inst,
+                                 hw::InstanceGraphBase &instanceGraph) {
```

Comment:
The PR could be a bit cleaner by putting creating a private member of the pass to store an `InstanceGraph *` as opposed to passing this into a member function.